test.py.in

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+import os
+import sys
+llvm_lit = os.path.join('@LLVM_TOOLS_BINARY_DIR@', 'llvm-lit.py')
+sys.argv = [llvm_lit, '@SwiftREPL_TESTS_DIR@', '-v', '-vv']
```

Comment:
Why the `-v` and `-vv` as default arguments?  You should be able to remove `test.py.in` and just do this from CMake itself.